Foundation/CPAttributedString.j

```diff
@@ -769,6 +771,11 @@
     }];
 }
 
++ (CPAttributedString)attributedStringWithAttachment//(CPTextAttachment)attachment
```

Comment:
this is not useful as is. we the attachment in order to generate an attributedStringWithAttachment

Reply:
I saw this morning that I accididentally mixed up and I already undo the commit in this branch.